src/gm3/components/layers/wms.js

```diff
@@ -46,7 +46,10 @@ function defineSource(mapSource) {
         url: mapSource.urls[0],
         // This is a carry over from previous generations behaviour.
         ratio: 1.0,
-        params: Object.assign({'LAYERS': layers.join(',')}, mapSource.params),
+        params: Object.assign({
+            'VERSION': '1.1.0',
```

Comment:
WMS version is really something that should be settable in the mapbook.  A server doesn't have to advertise/accept all the versions.  I don't know if it makes more sense to set this up as a wms vs wms13 layer type or via a parameter.